libs/sdk-react-native/ios/BreezSDKMapper.swift

```diff
@@ -455,7 +470,44 @@ class BreezSDKMapper {
             "message": messageSuccessActionData.message
         ]
     }
-    
+
+    static func dictionaryOf(receivePaymentRequest: ReceivePaymentRequest) -> [String: Any] {
+        return [
+            "amountSats": receivePaymentRequest.amountSats,
+            "description": receivePaymentRequest.description,
+            "preimage": arrayOf(preimageBytes: receivePaymentRequest.preimage),
+            "openingFeeParams": dictionaryOf(params: receivePaymentRequest.openingFeeParams)
+        ]
+    }
+
+    static func dictionaryOf(receivePaymentResponse: ReceivePaymentResponse) -> [String: Any] {
+        return [
+            "lnInvoice": dictionaryOf(lnInvoice: receivePaymentResponse.lnInvoice),
+            "openingFeeParams": dictionaryOf(params: receivePaymentResponse.openingFeeParams),
+            "openingFeeMsat": receivePaymentResponse.openingFeeMsat
+        ]
+    }
+
+    static func dictionaryOf(receiveOnchainRequest: ReceiveOnchainRequest) -> [String: Any] {
+        return [
+            "openingFeeParams": dictionaryOf(params: receiveOnchainRequest.openingFeeParams)
+        ]
+    }
+
+    static func dictionaryOf(buyBitcoinRequest: BuyBitcoinRequest) -> [String: Any] {
+        return [
+            "provider": asBitcoinProvider(provider: buyBitcoinRequest.provider),
+            "openingFeeParams": dictionaryOf(params: buyBitcoinRequest.openingFeeParams)
+        ]
+    }
```

Comment:
This is not needed, but an asBuyBitcoinRequest function should be added to convert a dictionary to BuyBitcoinRequest.

Reply:
Done as part of a1b1947